AnkiDroid/src/main/java/com/ichi2/anki/FieldEditText.kt

```diff
@@ -171,7 +172,17 @@ class FieldEditText : FixedEditText, NoteService.NoteField {
         return if (mediaUri == null) {
             false
         } else {
-            pasteListener!!.onPaste(this, mediaUri, description)
+            try {
+                pasteListener!!.onPaste(this, mediaUri, description)
+            } catch (ex: NullPointerException) {
+                Timber.w(ex, "Failed to paste media")
+                showSnackbar(context.getString(R.string.multimedia_editor_something_wrong))
+                false
+            } catch (e: Exception) {
+                Timber.w(e, "Failed to paste media")
+                showSnackbar(context.getString(R.string.multimedia_editor_something_wrong))
+                false
+            }
```

Comment:
these are functionally identical blocks now aren't they? propose collapse

```suggestion
            } catch (e: Exception) {
                Timber.w(e, "Failed to paste media")
                showSnackbar(context.getString(R.string.multimedia_editor_something_wrong))
                false
            }
```

Reply:
Earlier this was sending report to ACRA, my bad will fix it